src/directions/main.ts

```diff
@@ -943,7 +943,7 @@ export default class MapLibreGlDirections extends MapLibreGlDirectionsEvented {
    * Sets the waypoints' bearings values. Does not produce any effect in case the `bearings` configuration option is
    * disabled.
    */
-  set waypointsBearings(bearings: [number, number | undefined][]) {
+  set waypointsBearings(bearings: ([number, number] | undefined)[]) {
```

Comment:
Would the newly-added `Position` type suit here?

Reply:
Position allows for three-dimensional coordinates so having this would be more precise.